{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/openapikit\/validationerror\/reason"]}],"primaryContentSections":[{"declarations":[{"tokens":[{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"text":"reason","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"}],"platforms":["Linux"],"languages":["swift"]}],"kind":"declarations"}],"kind":"symbol","abstract":[{"type":"text","text":"The reason for the validation failure."}],"hierarchy":{"paths":[["doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit","doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError"]]},"schemaVersion":{"major":0,"minor":3,"patch":0},"metadata":{"modules":[{"name":"OpenAPIKit"}],"externalID":"s:10OpenAPIKit15ValidationErrorV6reasonSSvp","title":"reason","symbolKind":"property","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"reason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"role":"symbol","roleHeading":"Instance Property"},"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError\/reason"},"references":{"doc://OpenAPIKit/documentation/OpenAPIKit":{"role":"collection","type":"topic","abstract":[],"title":"OpenAPIKit","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit","kind":"symbol","url":"\/documentation\/openapikit"},"doc://OpenAPIKit/documentation/OpenAPIKit/ValidationError":{"type":"topic","navigatorTitle":[{"kind":"identifier","text":"ValidationError"}],"kind":"symbol","abstract":[{"text":"Validation errors are just a textual reason for validation failure and","type":"text"},{"text":" ","type":"text"},{"text":"a coding path where the validation error occurred.","type":"text"}],"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ValidationError"}],"title":"ValidationError","url":"\/documentation\/openapikit\/validationerror","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError","role":"symbol"},"doc://OpenAPIKit/documentation/OpenAPIKit/ValidationError/reason":{"fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"reason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"kind":"symbol","title":"reason","role":"symbol","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError\/reason","url":"\/documentation\/openapikit\/validationerror\/reason","type":"topic","abstract":[{"text":"The reason for the validation failure.","type":"text"}]}}}